dashboard/app/helpers/levels_helper.rb

```diff
@@ -126,6 +125,12 @@ def app_options
       callouts: callouts
     )
 
+    # External project levels are any levels of type 'external' which use
+    # the projects code to save and load the user's progress on that level.
+    view_options(is_external_project_level: true) if @level.pixelation?
```

Comment:
projects does still need isExternalProjectLevel to determine (1) whether to display headers and (2) whether to load the data on the server from appOptions.channel. In case (2), the existing behavior was not to load from appOptions.channel because we can assume that the channel is empty in that exact case. grep the PR diff for isExternalProjectLevel to see which places I'm talking about.